src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
Advantages both ways. Rewriting the file at every boot can be a reasonable prophylactic covering any number of future coding errors. However, in a security audit you generally have to explain the "why" behind each modified file, which is a pain. If the checksum of the generated file is the same each and every time, I would say writing the file at boot is okay. However, if the checksum can change randomly, then I would think the effort to ensure the correctness of the write on demand the code is worth it.